ProjectLighthouse.Servers.GameServer/Controllers/Slots/SlotsController.cs

```diff
@@ -278,7 +280,9 @@ public async Task<IActionResult> LuckyDipSlots([FromQuery] int pageStart, [FromQ
 
         GameVersion gameVersion = token.GameVersion;
 
+        DateTimeOffset cutoff = DateTimeOffset.Now.AddDays(-31);
         List<SlotBase> slots = (await this.database.Slots.ByGameVersion(gameVersion, false, true)
+            .Where(s => s.FirstUploaded >= cutoff.ToUnixTimeMilliseconds())
```

Comment:
I think this could be handled better by having them sorted randomly but biased so that newer levels are more likely to show up. This way levels will still show up even if there haven't been any new uploads in the past month.